pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -640,6 +655,30 @@ func getLastDrainTimeStamp(pdbStateMap *corev1.ConfigMap, key string) (time.Time
 	return lastDrainTimeStamp, nil
 }
 
+func (r *ReconcileClusterDisruption) getAllowedDisruptions(pdbName, namespace string) (int32, error) {
+	usePDBV1Beta1, err := k8sutil.UsePDBV1Beta1Version(r.context.ClusterdContext.Clientset)
+	if err != nil {
+		return 0, errors.Wrap(err, "failed to fetch pdb version")
```

Comment:
It's a bit confusing to return 0 on an error where 0 is also a valid value. Can we return `-1` instead?